client/src/components/Sidebar/SideBar.tsx

```diff
@@ -20,22 +20,22 @@ const Sidebar = (): JSX.Element => {
     >
       <div className={classes.toolbar} />
       <List>
-        <ListItem button component={Link} to="/editprofile">
+        <ListItem button component={Link} to="/manageprofile/editprofile">
```

Comment:
Please change route names to hyphenated case. Ditto throughout :)

Eg: `/manage-profile/edit-profile`